### PR TITLE
Joe unit tests 2

### PR DESCRIPTION
**Pull request by JosiahDeYong**

### CODE REVIEW

File: TAScheduler/UnitTests/tests.py [hunk 1]
```diff
@@ -727,6 +727,216 @@ def test_max_capacity_instr(self):
...
+        tmpta.save()
+        self.admin.sectionTAAsmgt(tmpta, lecobj)
+        self.assertEquals(Lab.objects.get(ta=tmpta), lecture, "Did not assign correct laboratory")
+
+    def test_bad_ta(self):
+        laboratory = Lab.objects.create(
+            section_id=self.section
+        )
+        labobj = LabObj(laboratory)
+        labobj.save()
```
[the remark this answers is not in the document]
Comment (KevinMonahanUWM) [reply TAScheduler/UnitTests/tests.py:885]: Correct

File: TAScheduler/UnitTests/tests.py [hunk 2]
```diff
@@ -727,6 +727,216 @@ def test_max_capacity_instr(self):
...
+        self.assertRaises(self.admin.sectionTAAsmgt("String!", labobj), TypeError, "Does not raise typerror for bad TA")
```
[the remark this answers is not in the document]
Comment (KevinMonahanUWM) [reply TAScheduler/UnitTests/tests.py:886]: Any assertRaises tests should be done with the "with" statement

**Review by KevinMonahanUWM — COMMENTED** [review]
These look good to me, but we will know for sure once you write the methods

File: TAScheduler/UnitTests/tests.py [hunk 3]
```diff
@@ -727,6 +727,216 @@ def test_max_capacity_instr(self):
...
+        self.tempInstr = InstructorObj(self.hold_instr)
+
+        self.hold_course = Course.objects.create(
+            course_id=101,
+            semester='Fall 2023',
+            name='Introduction to Testing',
+            description='A course about writing tests in Django.',
+            num_of_sections=3,
+            modality='Online',
+            credits=4
```
[the remark this answers is not in the document]
Comment (AlecBrookens) [reply TAScheduler/UnitTests/tests.py:770]: CREDIT BAD

**Review by AlecBrookens — APPROVED** [review]

**Comment by AlecBrookens** [comment]
assuming all comments are resolved, looks good

**Review by Kiran-Ee — APPROVED** [review]
Looks great, thanks for making those changes !

**Pull request merged**